### ISSUE TITLE
Search not working on bigint fields

### ISSUE THREAD

**Issue by mashirozx** [nocodb/nocodb#589]
**Please enter the following details**

Copy and paste project info : [Youtube Video](https://www.youtube.com/watch?v=AUSNN-RCwhE)

or 

please provide the following details :
NocoDB used as docker : true
NocoDB version : 0.11.39
Database used in NC_DB URL  : pg
Project was created  by clicking : New Project by connecting to external database
Database on which spreadsheet is created : pg 
OS on which NocoDB is running : the official docker image, host on ubuntu 20.04
Node.js version if running as node : 
Database version : I don't know... Just using a docker-compose file...

PS. cannot copy project info:
`/dashboard/?q=sqlOp_ncProjectInfo`, which returns `{"msg":"Not allowed"}`

**Steps To Reproduce**
1. use this docker compose file: 
```yml
version: '3.7'
services:
  nocodb_pg:
    image: postgres
    restart: always
    volumes:
      - ./nocodb_pg_data:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: password
      POSTGRES_USER: postgres
      POSTGRES_DB: root_db
    healthcheck:
      test: pg_isready -U "$$POSTGRES_USER" -d "$$POSTGRES_DB"
      interval: 10s
      timeout: 2s
      retries: 10
    networks:
      - internal_network

  nocodb:
    depends_on:
      nocodb_pg:
        condition: service_healthy
    image: nocodb/nocodb:latest
    ports:
      - "2080:8080"
      - "2081:8081"
      - "2082:8082"
      - "2083:8083"
    restart: always
    networks:
      - external_network
      - internal_network
    environment:
      NC_DB: "pg://nocodb_pg:5432?u=postgres&p=password&d=root_db"

networks:
  external_network:
  internal_network:
    internal: true
```

**Expected behavior**
Search records by id, just like `SELECT * FROM statuses WHERE id = 107013263901678012;`

But the search feature seems not work for me, shown as below

**Screenshots**

Table structure:
![image](https://user-images.githubusercontent.com/16148054/135208872-f3c51aff-aed5-4aa0-958b-22bb46424b54.png)

Request:
![image](https://user-images.githubusercontent.com/16148054/135208588-e7194a53-5ef7-49cc-8c8b-0c79ad692853.png)

Response body：
```
{"msg":"select \"statuses\".\"id\" as \"Id\", \"statuses\".\"uri\" as \"Uri\", \"statuses\".\"text\" as \"Text\", \"statuses\".\"created_at\" as \"CreatedAt\", \"statuses\".\"updated_at\" as \"UpdatedAt\", \"statuses\".\"in_reply_to_id\" as \"InReplyToId\", \"statuses\".\"reblog_of_id\" as \"ReblogOfId\", \"statuses\".\"url\" as \"Url\", \"statuses\".\"sensitive\" as \"Sensitive\", \"statuses\".\"visibility\" as \"Visibility\", \"statuses\".\"spoiler_text\" as \"SpoilerText\", \"statuses\".\"reply\" as \"Reply\", \"statuses\".\"language\" as \"Language\", \"statuses\".\"conversation_id\" as \"ConversationId\", \"statuses\".\"local\" as \"Local\", \"statuses\".\"account_id\" as \"AccountId\", \"statuses\".\"application_id\" as \"ApplicationId\", \"statuses\".\"in_reply_to_account_id\" as \"InReplyToAccountId\", \"statuses\".\"poll_id\" as \"PollId\", \"statuses\".\"deleted_at\" as \"DeletedAt\", \"statuses\".\"local_only\" as \"LocalOnly\", \"statuses\".\"content_type\" as \"ContentType\", \"statuses\".\"quote_id\" as \"QuoteId\" from \"statuses\" where \"statuses\".\"id\" like $1 limit $2 - operator does not exist: bigint ~~ unknown"}
```

Join our discord : https://discord.gg/5RgZmkW for realtime help.

Node version : **inside latest official docker**
Arch type: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
packageVersion: **0.11.39**

**Comment by johnhorsema** [comment]
Fixed this in pull request #636.

**Comment by marco-santulli** [comment]
hi I am getting this same error using postgres during searches on a bigint field. maybe there is a regression?

M

**Comment by wingkwong** [comment]
Fixed in ``0.84.12``.
